SignUp/SignUp/ThirdViewController.swift

```diff
@@ -28,17 +33,40 @@ class ThirdViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        pnumField.text = UserInformation.shared.pnum
```

Comment:
```suggestion
        phoneNumberField.text = UserInformation.shared.phoneNumber
```
줄임말 보다는 의미가 명확한 변수명을 사용하는것이 좋습니다